README.md

```diff
@@ -88,6 +113,28 @@ To deploy PromDash behind a reverse proxy you can set a global path prefix
 using the environment variable `PROMDASH_PATH_PREFIX`. Once set all URLs will
 start with the given prefix.
 
+One simple way to secure your Prometheus deployment with authentication is running a reverse proxy, Prometheus and PromDash on the same machine. The goal is to have a publicly available deployment for authenticated users via your reverse proxy while restricting raw `IP:PORT` access. We wont go cover any details on how to configure your reverse proxy, it depends on what you are using.
+
+**Reverse proxy** (eg. [nginx](http://nginx.org/), [HAProxy](http://www.haproxy.org/))
+* Setup two domains. Route requests by the specific hostnames and reject any other requests.
+ * `http(s)://prometheus.example.com` to backend `localhost:9090`
+ * `http(s)://dash.example.com` to backend `localhost:3000`
+* Configure ACL eg. HTTP basic auth for both backends
+
+**Run Prometheus** with binding only to localhost to restrict raw `IP:PORT` access that would circumvent your auth. Remember to pass `-web.external-url`. Example for docker: ([full reference](http://prometheus.io/docs/introduction/install/#using-docker))
```

Comment:
The meaning of "raw" is not obvious. How about:

"Run Prometheus with binding only to `localhost` to restrict direct external access over the network that would circumvent your authentication."